tonic-build/src/prost.rs

```diff
@@ -388,3 +447,53 @@ impl Builder {
         Ok(())
     }
 }
+
+#[derive(Debug, Default, Clone)]
+pub struct Attributes {
```

Comment:
does this need to live in the `prost` module?

Reply:
nope! do you have a preferred module? otherwise i'll move it to the crate root

Reply:
lib rs is probably fine